tools/static-assets/server/server-json.js

```diff
@@ -1,4 +1,6 @@
 var fs = require("fs");
 var path = require("path");
 var serverJsonPath = path.resolve(process.argv[2]);
-module.exports = JSON.parse(fs.readFileSync(serverJsonPath, 'utf8'));
+module.exports = JSON.parse(
+  fs.readFileSync(serverJsonPath, 'utf8').normalize('NFC')
```

Comment:
Exactly - we could always consider making the `unicodeNormalizePath` function a little more generic, so it's open to all strings. It wouldn't really fit in `mini-files.js` at that point though; it would belong in a more generic string helper location. I'm not sure where that would be in the context of the build tools though.